### DISCUSSION TITLE
Warm SST bias in MOM6 and Pacific precipitation

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#503] [Discussion about coupled runs]
This discussion is about the warm SST bias in MOM6 and Pacific precipitation

**Comment by adamrher** [comment]
Some plots comparing 54, 67, 69.

Std. dev. of PRECT (below):
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/6cf35906-6d80-429d-aa39-b308866e4aea)

Std. dev. of PRECC (below):
![temp_PRECC](https://github.com/NCAR/amwg_dev/assets/29961476/ed3bad08-0d0f-4f76-a322-286784404698)

Std. dev. of PRECL (below):
![temp_PRECL](https://github.com/NCAR/amwg_dev/assets/29961476/faf61bc7-0f5a-42b2-8874-f418db6bb7c3)

Mean of PRECL (below):
![temp_PRECL](https://github.com/NCAR/amwg_dev/assets/29961476/669259af-5dfb-4e68-bcd9-42a4ade70f70)

Mean of PRECC (below):
![temp_PRECC](https://github.com/NCAR/amwg_dev/assets/29961476/e05c6f5d-91f8-446b-94ba-9c2fe4507f32)

**Comment by adamrher** [comment]
54, 67, 69 but also including 78b for reference:

Mean of PRECT (below):

![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/37f14ccd-e769-431e-8b30-7a659bdc1aa7)

Mean of TMQ (below):

![temp_TMQ](https://github.com/NCAR/amwg_dev/assets/29961476/7ad37447-0091-4ce3-9329-134c19d9b8ed)

Mean of SST (below):

![temp_SST](https://github.com/NCAR/amwg_dev/assets/29961476/0cab4999-a9a2-4c56-8004-62cadac7caca)

**Comment by JulioTBacmeister** [comment]
Attached is a comparison showing PRECT in the latest coupled run I could find with POP - issue #24 (left) and the earliest coupled run I could find with MOM - issue #115 (right).  

I argue that the dry oval we are seeing in the equatorial east Pacific that we are seeing came in here.  So either with MOM or with the atmosphere changes between #24 and #115.   

![image](https://github.com/NCAR/amwg_dev/assets/9221710/3d298959-69ac-4765-b503-e8dc7113624f)

**Reply by JulioTBacmeister** [reply]
Of course I'm inclined to 'blame' MOM

**Reply by adamrher** [reply]
Interesting lead Julio. Any chance this run https://github.com/NCAR/amwg_dev/issues/24 still exists?

[edit - it does exist. 
```
/glade/campaign/cgd/amp/amwg/runs/b.e21.BWsc1850.ne30_L48_BL10_cesm2_3_alpha05c_cam6_3_028_cam6_parcel_zm.004_zm2.hf
```
]

**Reply by adamrher** [reply]
Confirming the dry slot is much improved [edit - no, not improved. let's stick with it's different] in https://github.com/NCAR/amwg_dev/issues/24 (bottom panel - POP-L58):

Std. dev. of PRECT (below):
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/e6b9e6b7-afa7-4008-9b5a-f53143985beb)
Mean of PRECT (below):
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/ba97574f-4ce7-4e7f-8499-32809137a9d5)
Actually the dry slot might be considered worse in this older run. Do we have a L32 POP run from around this same time? I've always suspected this is related to our higher vertical resolution, so I'd like to confirm.

**Reply by JulioTBacmeister** [reply]
@adamrher @swrneale 
Clearly the double ITCZ is worse in #24 than in CESM2 (and in some of the more recent runs) in the western warm pool, but the weird SH ITCZ extension all the way to S America (which really pops out in the STDDEV PRECT plot) is not there.

**Reply by JulioTBacmeister** [reply]
P.S. There was a 32 level B-case (#22)  run around the same time as #24. However, diags are not avaliable on the amwg_dev/issues page.  The case name is 

b.e21.BWsc1850.ne30_L32_cesm2_3_alpha05c_cam6_3_028_cam6_parcel_zm.001_zm2.hf. 

@cecilehannay Any idea whther the output is retrievable?

**Reply by cecilehannay** [reply]
/glade/campaign/cgd/amp/amwg/runs/b.e21.BWsc1850.ne30_L32_cesm2_3_alpha05c_cam6_3_028_cam6_parcel_zm.001_zm2.hf

**Reply by adamrher** [reply]
Updated the plots above to incl. this L32 run.

**Reply by adamrher** [reply]
Mean of SST (actually TS b/c L32 doesn't have SST; below):
![temp_TS](https://github.com/NCAR/amwg_dev/assets/29961476/07aa10d5-69af-49af-b1bd-2275c51d1e64)
Mean of TMQ (below):
![temp_TMQ](https://github.com/NCAR/amwg_dev/assets/29961476/ac010e1d-6dcb-40d1-bfbd-aa4b103cb37e)

**Reply by swrneale** [reply]
Adam could you do a difference plot set wrt to obs., or even CESM2?
And do I read it right that the TMQ RMSE jumps from 1.3 ish to 2+ ish??  Maybe re-evap will help here then?

**Reply by adamrher** [reply]
@swrneale here ya go (I'll add more fields as they come in). To be clear, MOM6 runs are the top two panels, POP2 the bottom two.

Mean of TS, difference with CESM2 (below):
![temp_TS](https://github.com/NCAR/amwg_dev/assets/29961476/727149f0-c3ef-48ce-bfef-f82b3e6f26df)
Mean of PRECT, difference with CESM2 (below):
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/05eb23a0-3c4a-49e6-8211-a6e5cfb946f5)
Mean of TMQ, difference with CESM2 (below):
![temp_TMQ](https://github.com/NCAR/amwg_dev/assets/29961476/af5e1ddd-9ba2-4570-a676-b36fbbcfd768)

**Reply by swrneale** [reply]
Yikes that is stark. I hadn't appreciated that red bullseye of warmer water in the SE Pacific with MOM. It's extremely local.

**Reply by adamrher** [reply]
Sure is. Is it worth looking at the surface energy budget terms over that bullseye?

**Reply by adamrher** [reply]
Mean of SWCF (below):
![temp_SWCF](https://github.com/NCAR/amwg_dev/assets/29961476/5a9b03ee-e9b1-4ec0-8dd1-ce31c54d69e0)
I suspect these too thin SWCF clouds are a response to the anomalously warm SSTs, rather than the other way around.

**Reply by swrneale** [reply]
OMG we really have to understand this. I wonder if MOM G-cases see hints of these types of differences? @gustavo-marques

**Comment by swrneale** [comment]
@JulioTBacmeister do you have the SSTs for the same plots layout?
But... given all the precip structure differences over land and around topography, one may not want to blame MOM as much (darn it!).

**Comment by JulioTBacmeister** [comment]
Thanks Cecile


On Tue, Mar 5, 2024 at 9:46 AM Cecile Hannay ***@***.***>
wrote:

>
> /glade/campaign/cgd/amp/amwg/runs/b.e21.BWsc1850.ne30_L32_cesm2_3_alpha05c_cam6_3_028_cam6_parcel_zm.001_zm2.hf
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/amwg_dev/discussions/503#discussioncomment-8682922>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACGLMTXW7MLRLJ5PYUEJRLTYWXZFTAVCNFSM6AAAAABEF5SQKKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMOBSHEZDE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*My working day may not be your working day. Please do not feel obliged to
reply to this message outside of your normal working hours.*

**Comment by adamrher** [comment]
@gustavo-marques here's hycom1 run 47 and 55, and corresponding default coord. runs 46 and 54.

TS differences from CESM2-piControl:
![temp_TS](https://github.com/NCAR/amwg_dev/assets/29961476/44851a9a-f5b2-437e-b66f-1f42c39cd291)
PRECT differences from the CESM2-piControl:
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/6ba8e111-b526-4db1-b656-825ca5c21994)
Looks like it cuts the differences in the SE Pacific from CESM2 in half. I'll look at some raw precip plots compared to obs next, as it's not clear if the bias has been reduced sufficiently wrt obs.

**Reply by adamrher** [reply]
PRECT fields in the runs. Generally it seems the bias is still there in these hycom1 runs, although not as severe.

Mean PRECT:
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/820e59a3-507f-41b2-b92f-0c48e1b64e56)
Std. dev. of PRECT:
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/d17297b8-9a27-4861-ad2f-eadca9bde5f1)

**Comment by JulioTBacmeister** [comment]
Looking at Jan of Year 1 in the last POP run and in "26", we already see a big difference in SWCF over the stratus regions:

![image](https://github.com/NCAR/amwg_dev/assets/9221710/471d9a77-a846-4e94-9baa-49a1360d7b10)

**Comment by JulioTBacmeister** [comment]
But looking at the SST vs SWCF relationship it looks generally similar in both runs:

![image](https://github.com/NCAR/amwg_dev/assets/9221710/a0f0ab18-52b1-4829-93ca-6a21ea9a6955)

I'd argue this shows the atmosphere in both runs is producing clouds in the same way according to the SSTs it is given

**Comment by gustavo-marques** [comment]
We have a 100-year simulation with CAM FV and MOM6 zstar:

`/glade/u/home/gmarques/campaign_MOM6/bmom.e23.f09_t061_zstar_N65.nuopc.GM_tuning.002/run`

@adamrher, @JulioTBacmeister, @cecilehannay @PeterHjortLauritzen 
Would it be worth checking if there is also an issue with SWCF/SST in this simulation?

**Reply by adamrher** [reply]
Here's what that run looks like (FV-MOM-L32) compared to other combinations of dycore, vertical grid and ocn model. 

TS difference with CESM2-piControl:
![temp_TS](https://github.com/NCAR/amwg_dev/assets/29961476/334d2c3c-eab1-4e61-b6f6-396535a8703d)
TMQ difference with CESM2-piControl:
![temp_TMQ](https://github.com/NCAR/amwg_dev/assets/29961476/735d1024-a2d7-4e4f-b562-759e8b5c0f7e)
SWCF difference with CESM2-piControl:
![temp_SWCF](https://github.com/NCAR/amwg_dev/assets/29961476/e0e91ce5-c166-42ff-88b0-28b58322129e)
LWCF difference with CESM2-piControl:
![temp_LWCF](https://github.com/NCAR/amwg_dev/assets/29961476/1f27e195-0b49-4944-b42e-9838365369c7)
Mean PRECT in the simulations:
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/26be9ade-12e3-44d8-8a1a-420cc603e6ea)
Std. dev. of PRECT in the simulations:
![temp_PRECT stddev](https://github.com/NCAR/amwg_dev/assets/29961476/fe6d7a1a-979c-4369-a71f-6d4b064f7e9a)

It's still problematic in this FV-MOM-L32 run. Although that E.Pac hot-spot (TS) is not as intense, the impact on precip is about the same as in 78b. The TS field is also warmer in a broader region of the Pacific.

**Reply by gustavo-marques** [reply]
Thank you, Adam!

**Reply by JulioTBacmeister** [reply]
Yes. Thanks Gustavo and Adam. This was a good check to have.

**Comment by gustavo-marques** [comment]
Here is an updated version of the plots presented during the last CAM meeting.

The map shows the two masks used to compute area-weighted SSTs: East Equatorial Pacific (green) and South East Equatorial Pacific (yellow). The two other plots show the area-weighted SST monthly climatology for these two regions. I averaged the forced cases and OISST between 1982 and 2020. For the fully coupled cases, I averaged between years 1 and 10.

### Comparison against OISST

For the East Equatorial Pacific region, BMOM cases 054 and 059 are colder from June to December (this controls the overall colder bias), CESM2 PI is shifted, and all the G cases are warmer than OISST.

For the region of South East Equatorial Pacific, the CESM2 PI shows colder temperatures from October to May, while G cases are slightly warmer from May to December. BMOM cases are too warm, and BMOM + waves is particularly problematic.

![image](https://github.com/NCAR/amwg_dev/assets/11339137/8285e640-21cb-4db4-87ae-cc98c05a728e)
![image](https://github.com/NCAR/amwg_dev/assets/11339137/4d1d311a-3916-4f16-aa77-73bfd8fc56c0)
![image](https://github.com/NCAR/amwg_dev/assets/11339137/e5f5295f-b3ae-46eb-858e-9ca9a09fa6da)

**Reply by fobryan3** [reply]
Here is an analysis of the (20 year) mean annual cycle of the surface heat flux in forced (2024.004) and coupled (054) cases with the same physics in MOM6 compared to observations at the Stratus mooring in the SE Pacific (85W,20S). In both cases, the ocean model is not taking up enough heat (net surface heat flux is below the observations), despite the excessive SW in the coupled case. The heat loss from the non-solar terms is excessive in both cases, a consequence of the too high SST. This suggest a hypothesis that the ocean is not transporting heat away from the surface in this region efficiently enough. Ongoing work will look at the mixed layer integrated heat budget.
![SrfHeatFlxAnnCyc_2024 004](https://github.com/NCAR/amwg_dev/assets/47898629/3793f168-5c22-4000-87d6-95569a92dd19)
![SrfHeatFlxAnnCyc_054](https://github.com/NCAR/amwg_dev/assets/47898629/3f46c743-cf3a-48ce-a4b9-74d71c679b36)

**Comment by fobryan3** [comment]
More of the same, but now adding a recent f-case to the set so we can compare f- , g- and b-cases directly. We compare the three experiments to in situ surface flux observations. The observations (solid lines in the plots) are averaged over the full available record (2000-2021). Model results are dashed. The f-case is averaged over the full 1995-2006 period of the run. The g-case is averaged over the same calendar years as the f-case (1995-2006). It is forced by JRA55 reanalysis. The b-case (82b) is averaged over years 38-49 (the same model years as the g-case). 
<img width="798" alt="Screenshot 2024-04-11 at 2 03 15 PM" src="https://github.com/NCAR/amwg_dev/assets/47898629/3f4ca9a7-06c4-43da-8813-0ca63e8b75fb">

The SW flux from the g-case (JRA55) agrees very well with the in situ obs. However, even with realistic SW, it produces SST biased a little high, especially in austral summer. The f-case has a strong  O(50 W/m^2) positive bias in surface absorbed solar. In fact, compared to this particular b-case it is worse. Keep in mind this only for the single grid point closest to the mooring, but are we confident that the atm is not eroding the stratus deck on its own without help from biased SST?

**Reply by PeterHjortLauritzen** [reply]
Thanks for the analysis Frank!

Could you add the CESM piControl to your plot? 

@adamrher made the plots below showing the DJF SWCF bias for CESM2 and CESM3 
 for F- and B-cases; 

![temp_SWCF moor](https://github.com/NCAR/amwg_dev/assets/8782989/4459d7c3-f132-41ce-8306-ca8c22039392)

The CESM3 F-case does show larger biases off the West coast of South America but the B-cases for CESM2 and CESM3 look comparable ...

**Reply by adamrher** [reply]
Status update from our clubb_c8 runs forcing thicker clouds in the E. Pacific. The runs were able to reduce the incident shortwave in the vicinity of the moor; clubb_c8=5.0 has less incident shortwave than in CESM2. While this reduces the problematic precip band in this region, the precip variability is still much larger in this region than in CESM2. I have not looked at SSTs yet, but the precip suggests some type of structural ocean warming remains.

![temp_SWCF](https://github.com/NCAR/amwg_dev/assets/29961476/a854dc5b-39ea-4916-a270-14c09be07993)

![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/0da385ac-e6bb-4691-8245-6e3a01afd7ac)

**Comment by fobryan3** [comment]
Definitely seeing cooler SST in 88b and 89b compared to 82b. Only remaining significant positive biases in 89b are right at the Central/South American coasts.

![tos_err_082b_y1-10](https://github.com/NCAR/amwg_dev/assets/47898629/a16ee2d3-1603-4027-9e33-4a1e40b7a986)
![tos_err_088b_y1-10](https://github.com/NCAR/amwg_dev/assets/47898629/796ce023-a8ed-4abd-8adf-cbec7f0ec33a)
![tos_err_089b_y1-10](https://github.com/NCAR/amwg_dev/assets/47898629/7fbc409e-6540-4fd9-b61e-afd85d7d200a)
<img width="971" alt="Screenshot 2024-04-18 at 8 07 00 AM" src="https://github.com/NCAR/amwg_dev/assets/47898629/9d7021e4-6770-4855-87af-11a3e6135ac8">

The absorbed SW now close to mooring obs during austral summer but too low in winter.  Opposite previous cases which were too high in summer closer in winter. CESM2 piCntrl was a little low in JFM, then too high through rest of the year. :
<img width="969" alt="Screenshot 2024-04-18 at 8 07 17 AM" src="https://github.com/NCAR/amwg_dev/assets/47898629/3afa6ea8-b92c-404c-8e43-6e719a2b08f1">

**Comment by zarzycki** [comment]
Total driveby, but in some of [Kyle Nardi's Betacast (hindcast) runs](https://colinzarzycki.com/cpt/kyle/webpage.html) (currently being written up for JAMES), we saw a SWCF sensitivity to clubb_beta in the region of interest in this ticket. I kind of wrote it off as a nothingburger based on the unclear global signal.

![image](https://github.com/NCAR/amwg_dev/assets/10341827/4b3f680c-f6ff-4305-aed4-38307835d37c)

Here, @kylemnardi found that increasing clubb_beta (somewhat uniquely) dimmed low clouds off the W coast of Peru/Chile. May be worth running a sensitivity simulation alongside the c8 runs to see if any clubb_beta signal "sticks" in longer runs?

**Reply by adamrher** [reply]
Thanks @zarzycki, Ben showed me this result as well and I'm intrigued. I looked at the equations and I think clubb_beta is to thl/qt what clubb_gamma is to w'2, in that it is a parameter on the width of the pdf for thl/qt. I wonder if the sensitivity shown in that plot is the LWP changing or the cloud fraction changing, or if those two are intrinsically tied together. I think Brian's results today showed that they are tied together when you change c8.

**Comment by cecilehannay** [comment]
We have been talking about separating the discussion about the stratocumulus and the precip bias as right both are folded into the same discussion. 

I started a new discussion about the runs we started after last CAM7 meeting last week.
https://github.com/NCAR/amwg_dev/discussions/527

I hope it is ok.

**Comment by adamrher** [comment]
The 82b_L32 experiment is interesting, but it's also very puzzling. The SSTs are cooler than CESM2, but the anomalous precip in the E.Pacific is same or worse.

TS difference from CESM2:
![temp_TS](https://github.com/NCAR/amwg_dev/assets/29961476/72d823aa-aca4-4576-90f2-1a7d9a6b1a4f)

STDDEV(PRECT) raw field:
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/64925993-0d58-429c-8318-967ae47e2e09)

MEAN(PRECT) raw field:
![temp_PRECT](https://github.com/NCAR/amwg_dev/assets/29961476/078f0492-5feb-413a-adc9-e8d66d2100bd)

The only thing that has helped reduce the precip bias so far is increasing clubb_c8 in 88b and 89b. I suspect that in those runs, it is the westward expansion of the MarineSc into the warm blob that is cooling the region, and this reduces surface driven convection and (possibly?) low level convergence over the blob. So perhaps what we need to do is thicken the clouds on the Cu side of the MarineSc to Cu transition.

**Comment by adamrher** [comment]
Looking at this precip bias seasonally, MAM seems to have the peak bias:

![temp_PRECT MAM](https://github.com/NCAR/amwg_dev/assets/29961476/1f62b845-4d05-4519-a16b-dd997e175ad0)

This morning a few of use rec'd an email from John Dunne, which included feedback on this problem from Andrew Wittenberg and John Krasting, who shared work by H. Annamali. Annamali hypothesizes that the excessive precip bias in the eastern Pacific -- primarily over land in Centeral and S.America -- may be driving westward propagating gravity waves that lead to anomalous subsidence and create that "dry slot" out in the central Pacific. The run L32.cam6 is consistent with this hypothesis, in that there is less bias precip in the east, and the dry slot not as severe. However, the SPCZ gets worse, extending to the S.American continent. This precip "bridge" actually contracts further north, as it was previously crouched out by the dry slot being so large.
